src/actions/users.js

```diff
@@ -376,31 +412,52 @@ export function getStatusesByIds(userIds) {
 
 export function getSessions(userId) {
     return bindClientFunc(
-        Client.getSessions,
+        Client4.getSessions,
         UserTypes.SESSIONS_REQUEST,
         [UserTypes.RECEIVED_SESSIONS, UserTypes.SESSIONS_SUCCESS],
         UserTypes.SESSIONS_FAILURE,
         userId
     );
 }
 
-export function revokeSession(id) {
-    return bindClientFunc(
-        Client.revokeSession,
-        UserTypes.REVOKE_SESSION_REQUEST,
-        [UserTypes.RECEIVED_REVOKED_SESSION, UserTypes.REVOKE_SESSION_SUCCESS],
-        UserTypes.REVOKE_SESSION_FAILURE,
-        id
-    );
+export function revokeSession(userId, sessionId) {
+    return async (dispatch, getState) => {
```

Comment:
Oh it's because I want to use `sessionId` when I dispatch the action and `bindClientFunc` only attaches the data from the driver response (which doesn't have the session id)